assignment2.yaml

```diff
@@ -0,0 +1,132 @@
+swagger: "2.0"
+info:
+  description: "Welcome to the swapi, the Star Wars API! This documentation should help you familiarise yourself with the resources available and how to consume them with HTTP requests. If you're after a native helper library then I suggest you scroll down and check out what's available. Read through the getting started section before you dive in. Most of your problems should be solved just by reading through it."
+  version: ""
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/api"
+tags:
+- name: "Films"
+  description: "A Film resource is a single film."
```

Comment:
`externalDocs` is not meant to be removed.
`externalDocs` is a URL where we can find more details about the tag.
```
externalDocs:
    description: "Find out more"
    url: "http://swapi.co/api/films/1/"
```
The question is still unanswered. is this url `API` or `Documentation`?